pkg/costmodel/costmodel.go

```diff
@@ -1095,7 +1095,7 @@ func (cm *CostModel) GetNodeCost(cp costAnalyzerCloud.Provider) (map[string]*cos
 				newCnode.GPU = fmt.Sprintf("%d", 0)
 			}
 			newCnode.VGPU = fmt.Sprintf("%d", q.Value())
-
+			gpuc = float64(q.Value())
```

Comment:
I think this makes sense to me. This conditional was not previously setting `gpuc` which would later affect calculations of gpuToRam and RamToCpu cost ratios.

Reply:
Just to confirm, you're currently hitting this bug because you have the following two labels on your node?

```yaml
    nvidia.com/gpu.count: '1'
    nvidia.com/gpu.replicas: '1'
```